config.json

```diff
@@ -1183,7 +1654,10 @@
         "slug": "gigasecond",
         "name": "Gigasecond",
         "uuid": "287331a4-64d5-4b35-b4c1-a1171dc338e1",
-        "prerequisites": [],
+        "prerequisites": [
+          "integers",
+          "erlang-libraries"
```

Comment:
I have seen the `:calendar` module pop up in a few exercises, especially this one. I personally haven't used before and haven't even heard about it until now. I created an issue to mention this module in the `erlang-libraries` concept (https://github.com/exercism/elixir/issues/584) but maybe we need a separate concepts for dealing with date and time in general?

Reply:
At least two practice exercises need knowledge about `:calendar` and the Elixir date/time modules,  so I created an issue to create a concept for them: https://github.com/exercism/elixir/issues/586